### DISCUSSION TITLE
Can't create an ID columns with numbers

### DISCUSSION THREAD

**Discussion by Olyno** [supabase/supabase#321] [Questions] [answered]
Hi, I'm really new with Supabase and I'm trying to create a really simple ``users`` table. Problem: I get an ``Unexpected token`` each time I'm trying to use an ``int`` or a ``float`` type, and the first ``id`` column is not created. Can you help me about it please?

*Don't care of the type in the gif, I missed the type of this one, but it still have the same issue 😅*

![Demo available at https://i.imgur.com/HVXINrS.gif](https://i.imgur.com/HVXINrS.gif)

With that I have another question: when should I use another type as primary key? (like uuid, text...)

**Comment by inian** [comment, marked as answer]
Hi @Olyno, thank you for reporting the bug. 

It should be fixed now. Can you try creating a new table?

**Reply by Olyno** [reply]
Yeay, it works now, thanks for your work and your reactivity!
